diff -up ./src/gldit/cairo-dock-keybinder.h.orig ./src/gldit/cairo-dock-keybinder.h
--- ./src/gldit/cairo-dock-keybinder.h.orig	2011-10-30 15:40:20.049122283 +0000
+++ ./src/gldit/cairo-dock-keybinder.h	2011-10-30 15:40:41.249816780 +0000
@@ -32,7 +32,6 @@
 #ifndef __CD_KEY_BINDER_H__
 #define __CD_KEY_BINDER_H__
 
-#include <glib/gtypes.h>
 #include "cairo-dock-struct.h"
 #include "cairo-dock-manager.h"
 
