--- CMakeLists.txt.link	2012-04-19 13:59:36.000000000 +0000
+++ CMakeLists.txt	2012-04-19 14:00:16.000000000 +0000
@@ -66,7 +66,7 @@
 
 ########### dependencies ###############
 
-set (packages_required "gthread-2.0 cairo librsvg-2.0 dbus-1 dbus-glib-1 libxml-2.0 xrender gl glu pangox libcurl")  # for the .pc
+set (packages_required "gthread-2.0 gmodule-2.0 cairo librsvg-2.0 dbus-1 dbus-glib-1 libxml-2.0 xrender gl glu pangox libcurl")  # for the .pc
 STRING (REGEX REPLACE " " ";" packages_required_semicolon ${packages_required})  # replace blank space by semicolon => to have more details if a package is missing
 pkg_check_modules ("PACKAGE" REQUIRED "${packages_required_semicolon}")
 
--- src/gldit/CMakeLists.txt.link	2012-04-19 14:10:45.000000000 +0000
+++ src/gldit/CMakeLists.txt	2012-04-19 14:11:32.000000000 +0000
@@ -100,12 +100,13 @@
 
 # Link the result to the librairies.
 target_link_libraries("gldi"
+	icon-factory
+	implementations
 	${PACKAGE_LIBRARIES}
 	${GTK_LIBRARIES}
 	${XEXTEND_LIBRARIES}
 	${LIBCRYPT_LIBS}
-	icon-factory
-	implementations)
+	dl)
 
 
 configure_file (${CMAKE_CURRENT_SOURCE_DIR}/gldi.pc.in ${CMAKE_CURRENT_BINARY_DIR}/gldi.pc)
